src/libexpr/pos-idx.hh

```diff
@@ -2,12 +2,15 @@
 
 #include <cinttypes>
 
+#include "util.hh"
```

Comment:
Why is this included?
Can't seem to tell.

Reply:
Thanks for asking. It was for `hash_combine`, but that wasn't obvious at all.
I've split that into its own header, partly because "util" is a bad scope for a header, but also for the less subjective argument, from the commit message:

> Splitting it out immediately answers questions like [this], without increasing the number of compilation units.
>
> I did consider using boost::hash_combine instead, but it doesn't seem to be quite as capable, accepting only two arguments.